README.md

```diff
@@ -25,6 +25,14 @@ heroku local:run bin/rails reloadhints:aleph
 ```
 _note_: `reloadhints:aleph` requires ENV `ALEPH_HINT_SOURCE`
 
+To reload custom hints in a development environment:
+```
+heroku local:run bin/rails reloadhints:custom[https://www.dropbox.com/blah/blah]
```

Comment:
My thought process for going with a parameter here was something like:
* There should be a specific URL which the loader always looks at!
* That requires that we have a multi-person-writeable Dropbox folder, covering all the people who might have control over the hinting process, and that people update the same file (overwriting it in the same location) rather than uploading a new file.
* Ooooh, I don't trust humans.
* This is more likely to work in practice if we get the URL.

The AlephHint loader comes out of a different workflow, where 1) the data source is automated and 2) we have control over the resultant URL (I *also* have no idea how Dropbox generates URLs, hence no reason to be confident they'd be stable). So I am sensitive to your concerns about the loaders presenting a different interface, but I also think it might be worth it in lieu of the fact that the underlying workflows are in fact quite different.

(Also I think ideally we automate all the loaders which *do* present stable interfaces and then we don't have to think about it any more.)